src/auth/ducks/types.ts

```diff
@@ -46,8 +46,8 @@ export interface RefreshTokenResponse {
 
 export enum PrivilegeLevel {
   NONE = -1,
-  STANDARD = 0,
-  ADMIN = 1,
+  STANDARD = 'standard',
```

Comment:
Is the backend all uppercase? should this match?

Reply:
Not too sure but all uppercase doesn't work here (default in backend seems to be lowercase `standard`)